third_party/lua/luadebug.lua

```diff
@@ -808,6 +926,8 @@ if color_maybe_supported and not os.getenv("NO_COLOR") then
     COLOR_YELLOW = string.char(27) .. "[33m"
     COLOR_RESET = string.char(27) .. "[0m"
     GREEN_CARET = string.char(27) .. "[92m => " .. COLOR_RESET
+    GREEN_CARET_SYM = string.char(27) .. "[92m=>" .. COLOR_RESET
```

Comment:
Maybe reuse `GREEN_CARET_SYM` in `GREEN_CARET` as you did above?

Reply:
That's not useful (not simplifying things, but rather complicate) in this case.